types/types.go

```diff
@@ -101,10 +101,10 @@ type Package struct {
 	// 'package x' line.
 	Name string
 
-	// DocComments from doc.go, if any.
+	// The comment right above the package declaration in doc.go, if any.
 	DocComments []string
 
-	// Comments from doc.go, if any.
+	// All comments from doc.go, if any.
```

Comment:
> For example: https://godoc.org/k8s.io/gengo/args There's no doc.go there, and yet it extracts the package comments correctly.

That's not what DocComments is doing, according to my experiment, it just has "The comment right above the package declaration in doc.go, if any."

`Comments` contains "All comments from doc.go, if any.", and I agree with @lavalamp it doesn't make sense.

For backwards compatibility, i think the current PR is the best.